tennis-scheduler-web-app/src/components/StartPage.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { useState } from "react"
+import { useNavigate } from 'react-router-dom'
+
+export const StartPage = () => {
+
+
+    return (
+        <>
+        </>
```

Comment:
What is difference between start and home page?

Reply:
Home page is page for logged user, and start will be page with tennis courts

Reply:
Maybe to rename it to be more intuitive